src/frontend/components/SøknadsSteg/OmBarnet/OmBarnet.tsx

```diff
@@ -90,53 +97,63 @@ const OmBarnet: React.FC<{ barnetsId: BarnetsId }> = ({ barnetsId }) => {
                 />
             )}
 
-            {skjema.felter.borFastMedSøker.erSynlig && (
```

Comment:
Her er egentlig bare denne sjekken fjernet i tillegg til det to ja/nei spm lenger ned